idunn/blocks/covid19.py

```diff
@@ -1,45 +1,63 @@
+import logging
 from idunn import settings
+from idunn.utils.covid19_dataset import get_poi_covid_status
 from .base import BaseBlock
-from .opening_hour import OpeningHourBlock
+from .opening_hour import OpeningHourBlock, parse_time_block
 from pydantic import constr
-from typing import ClassVar, List, Optional
+from typing import ClassVar, Optional
+
+logger = logging.getLogger(__name__)
 
 
 class Covid19Block(BaseBlock):
     BLOCK_TYPE: ClassVar = "covid19"
     STATUSES: ClassVar = ["open_as_usual", "open", "maybe_open", "closed", "unknown"]
 
-    status: constr(regex="({})".format("|".join(STATUSES)))
+    status: constr(regex="^({})$".format("|".join(STATUSES)))
     opening_hours: Optional[OpeningHourBlock]
     note: Optional[str]
+    contribute_url: Optional[str]
+
+    @classmethod
+    def get_caresteouvert_link(cls, place):
```

Comment:
For readability reason, I think you should rename it "get_ca_reste_ouvert_link".

Reply:
done